### DISCUSSION TITLE
Intel QSV and watch folder

### DISCUSSION THREAD

**Discussion by johnvick** [jlesage/docker-handbrake#308] [Q&A]
I have an 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz and have never been able to get watch folder to work with QSV, but it does work with CPU encoding profiles. I am in the market to buy a new mini PC likely Chinese NUC clone by Topton (as I have at present). I think these and similar devices use the Intel laptop CPUs, but not 100% on this.

Question is - can anyone confirm they have QSV and watch folder working with Docker Handbrake and if so, with what CPU? I am favouring an Intel 13th gen i7 or i9, would be good if these worked.

I use non-docker Handbrake on a new Surface Laptop 5 i5 version and results are good with QSV, comparable to my desktop equipped with RTX 3070Ti encoding with Staxrip and NVENC for similar file sizes.

**Comment by jlesage** [comment]
> I have an 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz and have never been able to get watch folder to work with QSV

Were you able to use QSV with the GUI ?  The capability of using QSV should not be affected by the usage of the watch folder vs GUI.

**Reply by johnvick** [reply]
https://github.com/jlesage/docker-handbrake/issues/210

**Comment by johnvick** [comment]
Yes, QSV works fine just not with the watch folder. I opened a couple of threads on this earlier on, spent hours trying to get it to work, but no success.

**Comment by micahs72** [comment]
I know this is old and you've moved on by now, but for the sake of posterity: I had a problem very similar. For me, I could use auto-convert for 720qsv but not 1080qsv. I found a thread where their solution was putting this:  'AUTOMATED_CONVERSION_HANDBRAKE_CUSTOM_ARGS=--disable-qsv-decoding' 
in the handbrake compose file. (or just the '--disable-qsv-decoding' if you use docker run.) Apparently, the problem has to do with hardware **de**code -- The gui uses software decode for the qsv profiles but the auto-convert thing uses hardware decode. This command turns off the hw decoding and leaves software decoding & hw encoding. This is what worked for me. 

This workaround info should be more easily available since this is such a basic function that doesn't work properly. Maybe this should be turned off by default. hth

**Reply by johnvick** [reply]
Thanks, I came across this in another thread and it is working fine for me on three devices.

**Reply by Yxini-p** [reply]
> I know this is old and you've moved on by now, but for the sake of posterity: I had a problem very similar. For me, I could use auto-convert for 720qsv but not 1080qsv. I found a thread where their solution was putting this: 'AUTOMATED_CONVERSION_HANDBRAKE_CUSTOM_ARGS=--disable-qsv-decoding' in the handbrake compose file. (or just the '--disable-qsv-decoding' if you use docker run.) Apparently, the problem has to do with hardware **de**code -- The gui uses software decode for the qsv profiles but the auto-convert thing uses hardware decode. This command turns off the hw decoding and leaves software decoding & hw encoding. This is what worked for me.
> 
> This workaround info should be more easily available since this is such a basic function that doesn't work properly. Maybe this should be turned off by default. hth

I have a question；
Using Intel QSV to encode H265 video, GPU only participates in decoding and not in encoding，then the CPU remains in a fully loaded state
